charts/netbox/values.yaml

```diff
@@ -313,7 +317,7 @@ externalDatabase:
   username: netbox
   password: ""
   existingSecretName: ""
-  existingSecretKey: postgresql-password
+  existingSecretKey: postgres-password
```

Comment:
should this maybe default to `password`?

Reply:
Would look cleaner, but let's not change the API here